pkg/machine/fcos.go

```diff
@@ -18,32 +18,90 @@ import (
 	"github.com/coreos/stream-metadata-go/fedoracoreos"
 	"github.com/coreos/stream-metadata-go/release"
 	"github.com/coreos/stream-metadata-go/stream"
-
 	digest "github.com/opencontainers/go-digest"
 	"github.com/sirupsen/logrus"
 )
 
-// These should eventually be moved into machine/qemu as
-// they are specific to running qemu
-var (
-	artifact = "qemu"
-	Format   = "qcow2.xz"
-)
+type imageCompression int64
+type artifact int64
+type imageFormat int64
 
 const (
 	// Used for testing the latest podman in fcos
 	// special builds
 	podmanTesting     = "podman-testing"
 	PodmanTestingHost = "fedorapeople.org"
 	PodmanTestingURL  = "groups/podman/testing"
+
+	Xz imageCompression = iota
+	Zip
+	Gz
+	Bz2
+
+	Qemu artifact = iota
+	HyperV
+	None
+
+	qcow imageFormat = iota
+	vhdx
+	Tar
```

Comment:
Capitalize all for consistency? Unless there's a specific reason not to
```suggestion
	Qcow imageFormat = iota
	Vhdx
	Tar
```

Reply:
that happens in the next PR :)  Keeping it unexported rn